protobuf/mayastor.proto

```diff
@@ -179,6 +179,15 @@ message Replica {
   string uri = 6;   // uri usable by nexus to access it
 }
 
+// Replica space usage
+message ReplicaSpaceUsage {
+  uint64 capacity_bytes = 1; // replica capacity in bytes.
+  uint64 allocated_bytes = 2; // amount of actually allocated disk space for this replica in bytes
+  uint64 cluster_size = 3; // cluster size in bytes
```

Comment:
can replicas in the same pool have different cluster sizes? If not, would this good to have for pools?
At the moment we have a PoolType::Lvs, should we have a different usage depending on the pool type? Or do you think all of these would map into any type of pool, eg lvm: https://github.com/openebs/mayastor/pull/991

Reply:
Indeed, for SPDK blobstore, cluster size is property of blob store (Lvol), not blob. Probably, this can be different for other types of pools.
The reason I think it is good idea to have cluster size in Replica is that one won't need to do a second query to Pool to be able to calculate used size, etc.

Reply:
> The reason I think it is good idea to have cluster size in Replica is that one won't need to do a second query to Pool to be able to calculate used size, etc.

That's a good point.

> Indeed, for SPDK blobstore, cluster size is property of blob store (Lvol), not blob. Probably, this can be different for other types of pools.

In that case we could a oneOf, example:
```proto
message ReplicaSpaceUsage {
oneof lvs { ... }
}
```
or at least add a type to the replica, so we can tell it's lvs, but we can add that later as a new field to replica, and that way default we consider lvs?

Reply:
I think, it is better not to overload too much with potential future features.